src/languages/translations/en.json

```diff
@@ -0,0 +1,10 @@
+{
+  "layout": {
+    "title": "Title",
+    "subtitle": "Subtitle"
+  },
+  "button": {
+    "save": "Save",
+    "cancel": "Cancel"
+  }
+}
```

Comment:
please create a real example, so you can test if it actually works

Reply:
Completed